README.md

```diff
@@ -202,6 +202,8 @@ Thundra provides out-of-the-box instrumentation (tracing) for following librarie
 |mysql2              |`>=1.5.0`                  |
 |pg                  |`>=6.0.0`                  |
 |amqp 0.9.1          |`>=0.5.0`                  |
+|@google-cloud/pubsub|`>=1.2`                    |
```

Comment:
It might be better a simple Google PubSub example under `Integration Options for Containers and VMs` section